core/lib/dal/src/models/storage_sync.rs

```diff
@@ -38,7 +39,7 @@ pub(crate) struct SyncBlock {
     pub l1_batch_number: L1BatchNumber,
     pub last_in_batch: bool,
     pub timestamp: u64,
-    pub l1_gas_price: u64,
+    pub l1_gas_price: U256,
     pub l2_fair_gas_price: u64,
     pub fair_pubdata_price: Option<u64>,
```

Comment:
Why only type of `l1_gas_price` is changed?

Reply:
Yeah you're right, we will also update `l2_fair_gas_price` and `fair_pubdata_price`. Also I realized the type for prices need to be updated for sync blocks along with the protobuf `.proto` files used for external node sync messages (writing it here to track it)

Reply:
We have updated both `l2_fair_gas_price` and `fair_pubdata_price` and the `.proto` files, but I realized they are not actually used for syncing, as that's done through the regular RPC api the operator exposes.

Protobuf types were updated here [a478592](https://github.com/matter-labs/zksync-era/pull/1324/commits/a47859255385ed7cfedbd46bc96c9507cba3bfab) and a simple unit test for the change was  added here [2c75075](https://github.com/matter-labs/zksync-era/pull/1324/commits/2c750754c8ee8ab22e5d39c48289f43c6d6fd344), but to make sure: what is the whole protobuf/storagesync stuff used for? How do we know it's working outside of some basic unit tests?